packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.styles.ts

```diff
@@ -199,6 +199,9 @@ export const getStyles: IStyleFunction<IDropdownStyleProps, IDropdownStyles> = p
         right: 12,
         height: DROPDOWN_HEIGHT,
         lineHeight: DROPDOWN_HEIGHT - 2 // height minus the border
+      },
+      !disabled && {
+        cursor: 'pointer'
```

Comment:
Nice catch @redoz 👍 You killed it with this PR on noticing the details that matter to the overall user experience.
Thanks also for the other issues+PR pairs you are contributing to Fabric 🤝